test/python/WMCore_t/Services_t/DBS_t/DBSReaderReal_t.py

```diff
@@ -0,0 +1,485 @@
+#!/usr/bin/env python
+"""
+_DBSReader_t_
+
+Unit test for the DBS helper class.
+"""
+
+import unittest
+
+from Utils.PythonVersion import PY3
+
+from WMCore.Services.DBS.DBS3Reader import getDataTiers, DBS3Reader as DBSReader
+from WMCore.Services.DBS.DBSErrors import DBSReaderError
+from WMQuality.Emulators.EmulatedUnitTestCase import EmulatedUnitTestCase
+
+# A small dataset that should always exist
+DATASET = '/HighPileUp/Run2011A-v1/RAW'
+BLOCK = '/HighPileUp/Run2011A-v1/RAW#fabf118a-cbbf-11e0-80a9-003048caaace'
+FILE = '/store/data/Run2011A/HighPileUp/RAW/v1/000/173/657/B293AF24-BFCB-E011-8F85-BCAEC5329701.root'
+FILE2 = '/store/data/Run2011A/HighPileUp/RAW/v1/000/173/660/EE0B83F9-F3CB-E011-B996-BCAEC5329710.root'
+
+# A RECO dataset that has parents (also small)
+DATASET_WITH_PARENTS = '/Cosmics/ComissioningHI-PromptReco-v1/RECO'
+BLOCK_WITH_PARENTS = DATASET_WITH_PARENTS + '#7020873e-0dcd-11e1-9b6c-003048caaace'
+FILE1_WITH_PARENT = '/store/data/ComissioningHI/Cosmics/RECO/PromptReco-v1/000/180/841/368B76AA-4F09-E111-82CB-BCAEC5329721.root'
+FILE2_WITH_PARENT = '/store/data/ComissioningHI/Cosmics/RECO/PromptReco-v1/000/180/852/D82D6996-1B0B-E111-AA40-003048CF94A6.root'
+
+PARENT_DATASET = '/Cosmics/ComissioningHI-v1/RAW'
+PARENT_BLOCK = PARENT_DATASET + '#929366bc-0c31-11e1-b764-003048caaace'
+PARENT_FILE = '/store/data/ComissioningHI/Cosmics/RAW/v1/000/181/369/662EAD44-300C-E111-A709-BCAEC518FF62.root'
+
+
+#class DBSReaderTest(EmulatedUnitTestCase):
+class DBSReaderTest(unittest.TestCase):
+
+    def setUp(self):
+        """
+        _setUp_
+
+        Initialize the PhEDEx API to point at the test server.
+        """
+        self.endpoint = 'https://cmsweb-prod.cern.ch/dbs/prod/global/DBSReader'
+        self.testbed = 'https://cmsweb-testbed.cern.ch/dbs/int/global/DBSReader'
+        self.dbs = None
+        super(DBSReaderTest, self).setUp()
+        if PY3:
+            self.assertItemsEqual = self.assertCountEqual
+        return
+
+    def tearDown(self):
+        """
+        _tearDown_
+
+        """
+
+        super(DBSReaderTest, self).tearDown()
+        return
+
+    def testListDatatiers(self):
+        """
+        listDatatiers returns all datatiers available
+        """
+        self.dbs = DBSReader(self.endpoint)
+        prod = self.dbs.listDatatiers()
+        self.dbsTest = DBSReader(self.testbed)
+        testbed = self.dbsTest.listDatatiers()
+        print(prod)
```

Comment:
It will be much more clear if you'll add to print statement which instance results come from, e.g.
```
print("DBS {}\nresults {}".format(self.endpoint, prod))
```